codingtest_ajw/1일차/서울에서-김서방-찾기.js

```diff
@@ -0,0 +1,8 @@
+function solution(seoul) {
+    x=seoul.indexOf('Kim');
+    var answer = '김서방은 '+x+'에 있다';
+    return answer;
```

Comment:
```suggestion
    const index = seoul.indexOf('Kim');
    const answer = '김서방은 '+ index +'에 있다';
    return answer;
```
변수 네이밍의 경우엔 p,y찾기 코드에 피드백 드린 것 참고 하시면 될 것 같습니다.

아마 코테에서 기본적으로 answer를 var로 선언해주는 거로 알긴 한데 혹시나 해서 말씀 드리면 var는 사용을 지양하는 것이 좋습니다. 
그리고 변수를 선언 할 때 별다른 키워드(let, const, var)를 붙이지 않으면 암묵적으로 var로 선언이 되기 때문에 꼭 붙여주시는 것이 좋습니다.
https://bbubbush.tistory.com/47 
해당 내용 참고하시면면 좋을 것 같아요
지금 x나 answer의 경우 처음 선언될 때 이후로 값이 변경되지 않기 때문에 const로 선언 해주시면 될 것 같습니다.